solr/solrj-model/src/test/org/apache/solr/model/DummyTest.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.solr.model;
+
+import org.apache.solr.SolrTestCaseJ4;
+import org.junit.Test;
+
+public class DummyTest extends SolrTestCaseJ4 {
```

Comment:
Why was this added?

Reply:
I initially added it as a placeholder when creating the gradle submodule, so that I could make sure some tasks (i.e. `compileTest`, `test` worked.  But it shouldn't be here anymore, removing...